pkg/proc/fncall.go

```diff
@@ -12,13 +12,14 @@ import (
 	"sort"
```

Comment:
This file shouldn't be changed at all, the Go runtime has no support for injecting calls in non-amd64 architectures. Instead in proc_linux.go the functions Attach and Launch should be changed to pass false to NewCommonProcess when the architecture is arm64.